### ISSUE TITLE
Current top model on MTEB(eng, v2) has no documentation

### ISSUE THREAD

**Issue by KennethEnevoldsen** [embeddings-benchmark/mteb#2698]
<img width="851" alt="Image" src="https://github.com/user-attachments/assets/e4d506b3-9414-42e3-ab5a-30b86530a787" />

The model is available here: https://huggingface.co/bchoiced/CHAIN19

@Samoed I see that this was accepted in this [PR](https://github.com/embeddings-benchmark/results/pull/184). We probably shouldn't accept models with no available documentation on them

@24September, I am sorry for the confusion here, but is it possible that you can fill in the datasheet for this model? Otherwise, we will have to take it down.

Also I see that we haven't enquired on the training data. @24September can you please also confirm that you haven't trained on any of the training (or test set) of evaluated tasks.

**Comment by KennethEnevoldsen** [comment]
A similar case is true for:

<img width="834" alt="Image" src="https://github.com/user-attachments/assets/fad09424-eae6-4d4e-a14f-3fa044219008" />

Where the model also isn't available. I think we need to make this a requirement from now on (an API is fine, and we can accomodate for upcoming models, but then we should simply wait to merge)

**Comment by Samoed** [comment]
I've added it as is, because they're planning to share their report later, but I agree that we should add rules for this. Maybe create different PR template for adding a model with all rules filled

**Comment by KennethEnevoldsen** [comment]
So in addition to what is already there:

> - [ ] My model has a model sheet, report or similar
> - [ ] I solemnly swear that for all results submitted I have not training on the dataset including the training set. If I have I have disclosed it clearly.
> - [ ] My model is available, either as a publicly accessible API or publicly on e.g., Huggingface

**Comment by Samoed** [comment]
Yes, I think like that would be good

**Comment by isaac-chung** [comment]
I would go as far as reverting their PR until a paper /report has been shared. We should never accept "the report will come later" as an excuse. I'll believe it when I see it.

**Comment by Samoed** [comment]
I think we can just add flag to hide models from leaderboard
